bin/server.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+
+# LSST Data Management System
+# Copyright 2015 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+
+"""
+This is RESTful LSST Data Access Web Server. It handles /meta, /image, and /db
+
+@author  Jacek Becla, SLAC
+"""
+
+from flask import Flask, request
+from lsst.dbserv import dbREST_v0
+from lsst.imgserv import imageREST_v0
+from lsst.metaserv import metaREST_v0
+
+
+app = Flask(__name__)
+
+@app.route('/')
+def getRoot():
+    fmt = request.accept_mimetypes.best_match(['application/json', 'text/html'])
+    if fmt == 'text/html':
+        return ("LSST Web Service here. I currently support: "
+                "<a href='meta'>/meta</a>, "
+                "<a href='image'>/image</a>, "
+                "<a href='db'>/db</a.")
+    return "LSST Web Service here. I currently support: /meta, /image, /db."
+
+@app.route('/db')
+def getDb():
+    '''Lists supported versions for /db.'''
+    fmt = request.accept_mimetypes.best_match(['application/json', 'text/html'])
+    if fmt == 'text/html':
+        return "<a href='db/v0'>v0</a>"
+    return "v0"
```

Comment:
Is the intent here to return JSON object? Currently it returns HTML page with "v0" string on it.